op-node/p2p/rpc_server.go

```diff
@@ -327,7 +327,7 @@ func (s *APIBackend) ListBlockedAddrs(_ context.Context) ([]net.IP, error) {
 // Note: active connections to the IP subnet are not automatically closed.
 func (s *APIBackend) BlockSubnet(_ context.Context, ipnet *net.IPNet) error {
 	recordDur := s.m.RecordRPCServerRequest("opp2p_blockSubnet")
-	if ipnet == nil {
+	if ipnet == nil || ipnet.IP == nil || ipnet.Mask == nil {
 		log.Warn("invalid IPNet", "method", "BlockSubnet")
```

Comment:
While we are here, can the logs be directed to `s.log.Warn` (embedded service logger) instead of the global logger (package is named `log`). The global logs may just disappear in the void, and make parallel tests noisy.

Reply:
Changed all `log.` to `s.log` in this file